themes/default/content/docs/guides/scim/okta.md

```diff
@@ -100,6 +100,14 @@ Now that the SCIM connector knows how to connect to Pulumi, you need to adjust t
 5. Click **Save Mappings**.
 6. Click **Apply Updates**.
 
+## Adjusting the Okta to Pulumi Username Mappings
+
+It is important that usernames are not changed after a user is associated with a Pulumi application. To disable Okta from pushing username updates to Pulumi, perform the following steps:
+
+1. Under **Sign On** in the **Settings** module, click **Edit**.
+2. Under **Credentials Details**, change **Update application username on** from **Create and update** to **Create only**.
+3. Click **Save**.
```

Comment:
```suggestion
1. Under **Sign On** in the **Settings** module, select **Edit**.
2. Under **Credentials Details**, change **Update application username on** from **Create and update** to **Create only**.
3. Select **Save**.
```

we try and avoid using click in the docs as its pointer-centric. we also have a style guide with more details on what we try and align to: https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md